streams/delimiter_stream.ts

```diff
@@ -147,6 +150,7 @@ export class DelimiterStream extends TransformStream<Uint8Array, Uint8Array> {
             if (disposition === "prefix") {
               // Must keep last bytes of last chunk.
               bufs.push(last.subarray(lastSliceIndex));
+              chunkStart = 0;
```

Comment:
**comment:** Same thing here, this is unnecessary: The chunk start is necessarily already zero here but I decided to add this here for readability reasons.